README.md

```diff
@@ -30,7 +30,7 @@ The extension also adds a callback to the "ship" event on the `Shipment` model,
 
 ```ruby
   # config/initializers/easy_post.rb
-  Spree::EasyPost::configs[:purchase_labels?] = false
+  Solidus::EasyPost::configs[:purchase_labels?] = false
```

Comment:
This should be `CONFIGS` instead of `configs`, right?

Reply:
yes indeed! Good catch. update in a sec.